pkg/provisioning/service_test.go

```diff
@@ -577,7 +577,7 @@ func TestProvision_MultiplePipelines(t *testing.T) {
 
 	service := NewService(db, logger, pipelineService, connService, procService, "./test/pipelines3")
 	err := service.Init(context.Background())
-	is.NoErr(err)
+	is.True(err != nil) // duplicated pipeline id
```

Comment:
IIUC, this re-purposes the test for loading multiple pipelines, to actually check for duplicated. IMHO, it would be good to have both tests, i.e. checking if we can load multiple pipelines (happy path) and duplicates in different files.

Reply:
it still loads multiple pipelines and they are all created successfully {we made sure of that by the service.Expect() lines}, the only change will be this line, because no other methods will be expected to be called cuz the pipeline will be skipped.
I can easily copy it of course.. but lemme know if you still think it's better